test/conformance/event/event_adapter_level_zero_v2.match

```diff
@@ -1,12 +1,5 @@
-{{NONDETERMINISTIC}}
```

Comment:
`{{NONDETERMINISTIC}}` needs to stay.

Reply:
Why? The tests for v2 do not fail randomly anymore (at least it looks like that)